### PR TITLE
Create !crisis, !emergency and !mentalhealth

### PR DESCRIPTION
**Pull request by d-lord**
Both do the same thing - provide a list of crisis resources for quick reference.

Happy to add any other resources that may be of use :)

### CODE REVIEW

**Comment by TRManderson** [comment]
Good guy dal strikes again ❤️

**Comment by bradleysigma** [comment]
Implements #175

File: uqcsbot/scripts/crisis.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from uqcsbot import bot, Command
+
+
+@bot.on_command("crisis")
+@bot.on_command("mentalhealth")
+def handle_crisis(command: Command):
+    '''
+    `!crisis` or `!mentalhealth` - Get a list of mental health crisis resources.
+    '''
+    response = """*Mental health/crisis resources*
```
Comment (bradleysigma) [inline uqcsbot/scripts/crisis.py]: For nicer indentation:

```suggestion
response = "\n".join(["*Mental health/crisis resources*",
		      "24/7 UQ Counselling and Crisis Line: 1300 851 998",
		      "Campus Security (emergency): 07 3365 3333",
		      "Campus Security (non-emergency): 07 3365 1234",
		      "Counselling Services: https://www.uq.edu.au/student-services/counselling-services",
		      "UQ Psychology Clinic: https://clinic.psychology.uq.edu.au/therapies-and-services",	    
		      "UQ Resources: https://about.uq.edu.au/campaigns-and-initiatives/mental-health"])
```

Comment (TRManderson) [reply uqcsbot/scripts/crisis.py]: Alternatively move the string outside the body of the function for nice indenting and nice text formatting

Comment (d-lord) [reply uqcsbot/scripts/crisis.py]: I like Tom's a bit more. `response`, `RESPONSE` or?

**Comment by TRManderson** [comment]
All caps is standard for constants

**Comment by d-lord** [comment]
I'm wondering if it should include `!emergency` too. Three aliases seems a bit excessive but I want it to be intuitive/accessible. Maybe change `!mentalhealth` to `!emergency`?

**Comment by d-lord** [comment]
Hey fam, I've actioned the review and also added `!emergency` as an alias. Would appreciate a re-review so we can merge and forget about it until it's needed 😄

**Comment by nicklambourne** [comment]
reopening

**Pull request merged**